packages/react/src/sdk.ts

```diff
@@ -1,7 +1,9 @@
 import type { BrowserOptions } from '@sentry/browser';
-import { init as browserInit } from '@sentry/browser';
+import { init as browserInit, setContext } from '@sentry/browser';
 import { applySdkMetadata } from '@sentry/core';
 
+import { version } from 'react';
```

Comment:
how nice of them to export this. I wish more packages did that.

Reply:
just to sanity check, is this exported in all react versions (*that we support)?

Reply:
yes it is